test/basic.js

```diff
@@ -8,6 +8,109 @@ export default ({
         app.push("hello world");
 
         return eq(await request(app)("/"), "hello world");
-    })
+    }),
+
+    it("should print the 'hello world' by given handler", async () => {
+        let app = noflow();
+        
+        app.push((ctx) => {
+            ctx.body = "hello world";
```

Comment:
I have changed the `ctx.body`, now the `body` is an function. This will make sure that all the members of the `ctx` are references other than simple value.